build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1471,6 +1471,10 @@ message ExecuteOperationMetadata {
   // [ByteStream.Read][google.bytestream.ByteStream.Read] to stream the
   // standard error from the endpoint hosting streamed responses.
   string stderr_stream_name = 4;
+
+  // Additional implementation-specific metadata to send along with the
+  // Operation.
+  google.protobuf.Any additional_metadata = 5;
```

Comment:
Can we make it repeated and call it auxiliary_metadata? That way it’s consistent with the one we have in ExecutedActionMetadata.

Reply:
Makes sense! Updated.